simple/dgram.c

```diff
@@ -76,39 +76,23 @@ static int init_fabric(void)
 
 static int send_recv()
 {
-	struct fi_cq_entry comp;
 	int ret;
 
 	if (opts.dst_addr) {
 		fprintf(stdout, "Sending message...\n");
 		sprintf(buf, "Hello from Client!");
```

Comment:
`fi_dgram` currently doesn't work as expected. This line should read: 
```
sprintf(tx_buf, "Hello from Client!");
```

It also doesn't report failure since it never checks the buffer. Should we add a really simple string comparison against the expected value? (In a different PR)